server/data/prisonerSearchStore/prisonerSearchStore.ts

```diff
@@ -0,0 +1,30 @@
+import type { Prisoner } from 'prisonerSearchApiClient'
+import { RedisClient } from '../redisClient'
+import logger from '../../../logger'
+
+const PRISONER = 'prisoner'
+
+export default class PrisonerSearchStore {
```

Comment:
The concept of a "store" class simply takes a redis client instance, then exposes methods to get/retrieve objects from redis
In this case we are storing and retrieving a `Prisoner` instance in redis